src/platform/plugins/shared/discover/kibana.jsonc

```diff
@@ -54,10 +54,10 @@
       "kibanaUtils",
       "kibanaReact",
       "unifiedSearch",
-      "savedObjects"
+      "savedObjects",
```

Comment:
It would be nice to omit this noop change. It'll likely be reversed by someone else's linter that doesn't add trailing `,`... I think not adding trailing `,` in these `.jsonc` is more standard.